src/main/resources/META-INF/frontend/fc-applayout/fc-menuitem.js

```diff
@@ -60,7 +65,16 @@ class MenuItem extends ThemableMixin(PolymerElement) {
 		<iron-collapse-button no-icons="true">
 			<paper-icon-item id="item" slot="collapse-trigger" role="option" disabled="[[disabled]]">
 				<iron-icon src="[[src]]" icon="[[icon]]" slot="item-icon"></iron-icon>
-				<span id="label">[[label]]</span>
+				<dom-if if="[[href]]" restamp>
+					<template>
+						<a router-link href="{{href}}" id="label" onclick="getRootNode().host.__closeDrawer()">[[label]]</a>
```

Comment:
I'm wondering if an import statement is needed for using router-link attribute

Reply:
What would be the missing import? https://vaadin.com/docs/flow/routing/tutorial-routing-navigation.html

Reply:
Sorry, I got confused, because I thought we were implementing [Router Component](https://vaadin.github.io/vaadin-router/vaadin-router/demo/#vaadin-router-getting-started-demos). Now for what I understand this implementation of router link is a first step towards that direction so let's go ahead with this.